client/state/account-recovery/reset/actions.js

```diff
@@ -1,27 +1,37 @@
 /**
  * Internal dependencies
  */
-import wpcom from 'lib/wp';
 import {
 	ACCOUNT_RECOVERY_RESET_OPTIONS_ERROR,
 	ACCOUNT_RECOVERY_RESET_OPTIONS_RECEIVE,
 	ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST,
 } from 'state/action-types';
 
-export function fetchResetOptions( userData ) {
-	return ( dispatch ) => {
-		dispatch( { type: ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST } );
-
-		wpcom.undocumented().accountRecoveryReset( userData ).getResetOptions()
-			.then( options => dispatch( { type: ACCOUNT_RECOVERY_RESET_OPTIONS_RECEIVE, options } ) )
-			.catch( error => dispatch( { type: ACCOUNT_RECOVERY_RESET_OPTIONS_ERROR, error } ) );
+const fetchResetOptions = ( userData ) => {
+	return {
+		type: ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST,
+		userData,
 	};
-}
+};
```

Comment:
fun fact! we can simplify the syntax here and get rid of some noise since all we're doing is returning an object…

```js
const fetchResetOptions = userData => ( {
	type: ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST,
	userData,
} );
```